autosaloon_main/autosaloon_api/serializers.py

```diff
@@ -2,7 +2,7 @@
 
 from rest_framework import serializers
 
-class SaloonSerializer(serializers.ModelSerializer):
+class SaloonSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
why use HyperlinkModelserializer to ModeSerializer.. can you explain your usecase